package.json

```diff
@@ -5,7 +5,7 @@
   "main": "app.js",
   "scripts": {
     "start": "node .",
-    "test": "NODE_ENV=test mocha",
+    "test": "NODE_ENV=test mocha --recursive",
```

Comment:
:+1: this occurred around the same time my nvm and other path variables got corrupted